### PR TITLE
init 2023-00031-ivanti

### PR DESCRIPTION
**Pull request by Ralphhorn**

### CODE REVIEW

File: _cases/2023/DIVD-2023-00031.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- Ivanti MobileIron
+versions:
+- EPMM before 11.8.1.1
+- EPMM before 11.9.1.1
+- EPMM before 11.10.0.2
+recommendation: Update your system to the latest patched version
+patch_status: Fully patched
+#workaround: n/a
+status : Open
+start: 2023-07-18
```
Comment (Lennaert89) [inline _cases/2023/DIVD-2023-00031.md:21]: is this correct?

File: _cases/2023/DIVD-2023-00031.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+end: 
+timeline:
```
Comment (Lennaert89) [inline _cases/2023/DIVD-2023-00031.md:23]: add that we're notifying through our "trusted information sharing partners".

File: _cases/2023/DIVD-2023-00031.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- start: 2022-07-25
+  end:
+  event: "Ivanti releases a security advisory for CVE-2023-35078"
+- start: 2023-07-26
+  end:
+  event: "DIVD starts notifying owners of vulnerable systems"  
+# You can set IPs to n/a when this case isn't about IPs (e.g. stolen credentials)
+---
+## Summary
```
Comment (Lennaert89) [inline _cases/2023/DIVD-2023-00031.md:32]: Toevoegen dat er exploitation in the wild is?

File: _cases/2023/DIVD-2023-00031.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+This vulnerability will allow an unauthenticated ttacker to remotely access the api and "if exploited, this vulnerability enables an unauthorized, remote (internet-facing) actor to potentially access users’ personally identifiable information and make limited changes to the server"
+
+
+## What you can do
+
+If your Ivanti MobileIron appliance hasn't been updated to a secure version, we strongly advise you to patch it. e
+
+
+## What we are doing
```
Comment (Lennaert89) [inline _cases/2023/DIVD-2023-00031.md:48]: add that we're notifying through our "trusted information sharing partners".

File: _cases/2023/DIVD-2023-00031.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+
+### {% cve CVE-2023-35078 %} - Remote Unauthenticated API Access Vulnerability
+
+This vulnerability will allow an unauthenticated ttacker to remotely access the api and "if exploited, this vulnerability enables an unauthorized, remote (internet-facing) actor to potentially access users’ personally identifiable information and make limited changes to the server"
+
+
+## What you can do
+
+If your Ivanti MobileIron appliance hasn't been updated to a secure version, we strongly advise you to patch it. e
```
Comment (Lennaert89) [inline _cases/2023/DIVD-2023-00031.md:45]: last e?

**Review by Lennaert89 — CHANGES_REQUESTED** [review]

File: _cases/2023/DIVD-2023-00031.md (hunk 1 above)
Comment (Ralphhorn) [reply _cases/2023/DIVD-2023-00031.md:21]: fixed

File: _cases/2023/DIVD-2023-00031.md (hunk 2 above)
Comment (Ralphhorn) [reply _cases/2023/DIVD-2023-00031.md:23]: implemented and added to timeline

File: _cases/2023/DIVD-2023-00031.md (hunk 3 above)
Comment (Ralphhorn) [reply _cases/2023/DIVD-2023-00031.md:32]: adding in text

**Review by Lennaert89 — APPROVED** [review]

**Pull request merged**